packages/overlays/src/OverlaysManager.js

```diff
@@ -113,7 +113,7 @@ export class OverlaysManager {
       .reverse()
       .forEach((ctrl, i) => {
         // eslint-disable-next-line no-param-reassign
-        ctrl.elevation = i + 1;
+        ctrl.elevation = `${i + 1}`;
```

Comment:
can it be fixed here: https://github.com/ing-bank/lion/blob/2b73c50f6e7ff9ef96122a74536e7c4df9b8553b/packages/overlays/src/OverlayController.js#L420 
?

Reply:
Yep, good idea, since the getter is already type number too.